### PR TITLE
Update README.md with the new steps to install the plugin (#14)

### PR DESCRIPTION
**Pull request by oscard0m**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,10 +16,16 @@ unzip BitBar-v1.9.1.zip && mv BitBar.app ~/Applications && open BitBar.app
 - Click on the BitBar icon
 - Set your favourite directory as Plugin Folder
 - Clone this repos into your plugin folder
-- Copy the script outside
+- Copy the script and the package.json on your Plugin Folder
```
Comment (stefanosaittamrf) [inline README.md:19]: maybe just clone the repo can be better no?

Comment (oscard0m) [inline README.md:19]: Right now the repo has more files: package.json, README.md... These will be interpret as plugin candidates by Bitbar

**Comment by oscard0m** [comment]
ping

**Comment by oscard0m** [comment]
Deprecated PR

**Pull request closed without merging**